scripts/netdep-graph.sh

```diff
@@ -1,14 +1,20 @@
 #!/bin/sh
 
 # Usage:
-#     CHROMIUM_SOURCE_PATH=$HOME/chromium/src ./netdep-graph.sh
+#     CHROMIUM_SOURCE_PATH=$HOME/chromium/src ./netdep-graph.sh ../artifacts.log
 #     open network-dependency-graph.png
 
 if [[ x"$CHROMIUM_SOURCE_PATH" == x ]]; then
```

Comment:
Do you mean changing the error message to "You must set the environment variable CHROMIUM_SOURCE_PATH"?